tests/cli/test_integration_datasets.py

```diff
@@ -189,14 +189,15 @@ def test_dataset_import_real_doi_warnings(runner, project, sleep_after):
 
     result = runner.invoke(cli, ["dataset", "ls"])
     assert 0 == result.exit_code, format_result_exception(result) + str(result.stderr_bytes)
-    assert "pyndl_naive_discr_v0.8.1" in result.output
+    assert "pyndl_naive_discr_v0.6.0" in result.output
```

Comment:
I fixed this in one of my PRs as well, but I used 0.8.2 as that's the newest one

Reply:
I'll resolve this once merging develop